openmetadata-ui/src/main/resources/ui/src/components/nav-bar/NavBar.tsx

```diff
@@ -76,7 +76,7 @@ const NavBar = ({
 
   const profilePicture = useMemo(
     () => AppState?.userDetails?.profile?.images?.image512,
-    [AppState]
+    [AppState?.userDetails?.profile?.images]
```

Comment:
Can we take the `AppState?.userDetails?.profile?.images` in constant and use them here?

Reply:
Done 👍